ExtraDry/ExtraDry.Blazor/Components/Dry/DryStaticFilter.razor.cs

```diff
@@ -0,0 +1,42 @@
+namespace ExtraDry.Blazor;
+public partial class DryStaticFilter<TItem> : ComponentBase, IExtraDryComponent
+{
+    public DryStaticFilter()
+    {
+        ViewModelDescription = new ViewModelDescription(typeof(TItem), this);
+    }
+
+    /// <inheritdoc cref="IExtraDryComponent.CssClass "/>
+    [Parameter]
+    public string CssClass { get; set; } = string.Empty;
+
+    /// <inheritdoc cref="IExtraDryComponent.UnmatchedAttributes" />
+    [Parameter(CaptureUnmatchedValues = true)]
+    public Dictionary<string, object>? UnmatchedAttributes { get; set; }
+
+    [Parameter]
+    public int? InitialHierarchyLevel { get; set; }
+
+    [Parameter]
+    public string? TextFilter { get; set; }
+
+    [Parameter]
+    public Dictionary<string,object>? PropertyFilter { get; set; }
+
+
+    /// <inheritdoc cref="DryPageQueryView.PageQueryBuilder" />
+    [CascadingParameter]
+    public QueryBuilder? QueryBuilder { get; set; }
+
+    private ViewModelDescription ViewModelDescription { get; set; }
```

Comment:
This looks like it could be removed as it's not being used.  Quite a bit of overhead if it's not necessary.

Reply:
Done